WooCommerce/Classes/ViewRelated/Dashboard/MyStore/ChartPlaceholderView.swift

```diff
@@ -31,6 +31,7 @@ private extension ChartPlaceholderView {
     ///
     func setupView() {
         backgroundColor = .listForeground
+        topStackView.backgroundColor = .listForeground
```

Comment:
Oh! Until iOS 13 stack views didn't have any renderable background-color, nice to see that changed on iOS 14!

Reply:
Ooh right, this is a new change in iOS 14! (a [blog post](https://useyourloaf.com/blog/stack-view-background-color-in-ios-14/) about it) That explains the bug, the stack view's background color is set to white in the xib